plugins/modules/notification/cisco_webex.py

```diff
@@ -10,99 +10,104 @@
 
 DOCUMENTATION = '''
 ---
-module: cisco_spark
-short_description: Send a message to a Cisco Spark Room or Individual.
+module: cisco_webex
+short_description: Send a message to a Cisco Webex Teams Room or Individual
 description:
-    - Send a message to a Cisco Spark Room or Individual with options to control the formatting.
+    - Send a message to a Cisco Webex Teams Room or Individual with options to control the formatting.
 author: Drew Rusell (@drew-russell)
 notes:
   - The C(recipient_id) type must be valid for the supplied C(recipient_id).
-  - Full API documentation can be found at U(https://developer.ciscospark.com/endpoint-messages-post.html).
+  - Full API documentation can be found at U(https://developer.webex.com/docs/api/basics).
 
 options:
 
   recipient_type:
     description:
        - The request parameter you would like to send the message to.
        - Messages can be sent to either a room or individual (by ID or E-Mail).
-    required: True
+    required: yes
     choices: ['roomId', 'toPersonEmail', 'toPersonId']
+    type: str
 
   recipient_id:
     description:
       - The unique identifier associated with the supplied C(recipient_type).
-    required: true
+    required: yes
+    type: str
 
-  message_type:
+  msg_type:
     description:
        - Specifies how you would like the message formatted.
-    required: False
     default: text
     choices: ['text', 'markdown']
+    type: str
+    aliases: ['type']
```

Comment:
The old option was called `message_type`, not `type`:
```suggestion
    aliases: ['message_type']
```